Encountify/Views/RegisterPage.xaml.cs

```diff
@@ -99,12 +165,11 @@ private bool VerifyPassword(string password, string passwordconfirm)
         {
             if (!ValidPassword(password) | !ValidPassword(passwordconfirm))
```

Comment:
It seems that this part is causing a weird bug:
when you enter password: asdQWE123!@#, confirm password: asdQWE123123
It shows password should contain Error "Password should contain at least one special symbol" instead of that password do not match. This fixes it.
```suggestion
            if (!ValidPassword(password))
```